docs/content/users/configuration/config.md

```diff
@@ -127,7 +127,7 @@ Example: `dbimage_extra_packages: ["less"]` will add the `less` package when the
 You can configure a [version constraint](https://github.com/Masterminds/semver#checking-version-constraints) for DDEV that will be validated against the running DDEV executable and prevent `ddev start` from running if it doesn't validate. For example:
 
 ```yaml
-ddev_version_constraint: '>=v1.23.0-alpha1'
+ddev_version_constraint: '>= v1.23.0-alpha1'
```

Comment:
Does this matter? Was the lack of space an error?

Reply:
It doesn't matter, I just noticed we have this space everywhere else, so for consistency and better readability I added it here.